ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -550,6 +550,9 @@ def platform(self) -> dict | None:
             # this value is required for the solution to work.
             raise AristaAvdMissingVariableError("For AutoVPN RRs and Pathfinders, 'data_plane_cpu_allocation_max' must be set")
 
+        if (struct_cfg := get(self.shared_utils.platform_settings, "structured_config")) is not None:
```

Comment:
not sure this is the right place to add this. @ClausHolbechArista should the key be read inside the `custom_structured_config` python module instead?

`raw_eos_cli` is being read in base though.

Reply:
We should *not* just run `.update()` since you would be updating too early, and on the wrong level. The structured config should be root-level.
Instead you have two options:
- Return a list of dicts in a new cached_property called `struct_cfgs()`.
- Read this in the custom_structured_config module instead.

Reply:
Added a new cached_property called struct_cfgs()